task-requests/details/script.js

```diff
@@ -410,14 +436,17 @@ const renderGithubIssue = async () => {
   );
 };
 const renderRejectButton = (taskRequest) => {
+  if (self_user.roles['super_user']) {
+    rejectButton.classList.remove('hidden');
+  }
```

Comment:
both the `reject` and `approve` buttons should follow the same role-based logic, where they are hidden or shown based on whether the user is a superuser.

Reply:
to make the user experience better i added the hidden class to the reject button initially and when the ui render we check if user is super_user then only remove the hidden class otherwise no need to show the reject button to the normal member